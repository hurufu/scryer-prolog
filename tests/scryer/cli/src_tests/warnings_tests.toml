args = [
    "-f",
    "--no-add-history",
    "src/tests/warnings.pl",
    "src/tests/warnings1.pl",
    "src/tests/warnings2.pl",
    "-g", "halt"
]
