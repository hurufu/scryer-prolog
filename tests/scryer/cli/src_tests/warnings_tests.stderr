% Warning: integer/1 is a constant source of bugs, use integer_si/1 from library(si) at line 4 of warnings.pl
% Warning: (b,c) attempts to re-define (,)/2 at line 9 of warnings.pl
